pandas/tests/indexes/test_numeric.py

```diff
@@ -919,6 +919,10 @@ def test_constructor(self):
         res = Index([1, 2**63])
         tm.assert_index_equal(res, idx)
 
+        idx = Index([-1, 2**63], dtype=object)
```

Comment:
FYI, `Series` is very lenient against negative numbers in `uint64` too:
~~~python
>>> Series([-1], dtype='uint64')
0    18446744073709551615
dtype: uint64
~~~